.ci/danger/dangerfile.ts

```diff
@@ -5,7 +5,7 @@ execSync('npm install lodash');
 
 const flatten = require('lodash/flatten');
 const intersection = require('lodash/intersection');
-const isEmpty = require('lodash/isEmpty');
+const isEmpty = require('lodash/isEmpty.js');
```

Comment:
Does the extension make sense here? This is a CommonJS module with CommonJS style imports (require).

Reply:
It doesn't hurt anything, and I figured we may as well be consistent.  But I can remove it if preferred.

Reply:
Agreed. Should we then also add the extension to the above two lodash imports?

Reply:
I added them there, as well as all the other lodash imports I could find, just for good measure.